Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticSystem.cs

```diff
@@ -17,24 +17,8 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
     /// This extensibility allows not just for 1, 2, and 3-D springs, but
     /// allows for 4-dimensional quaternion springs.
     /// </summary>
-    internal abstract class ElasticSystem<T>
+    public abstract class ElasticSystem<T>
```

Comment:
Moving internal to public so that developers can subclass to make their own ElasticSystem variants.

Reply:
Why not make this an interface at this point rather than an abstract class?

Reply:
Oh, good point.

Reply:
@RogPodge it's now an interface, thank you :)